src/graphorm/index.ts

```diff
@@ -222,14 +223,35 @@ const obj = new GraphORM({
     requiredColumns: ['createdAt', 'userId', 'role'],
```

Comment:
I checked and `sourceId` is not needed here since we never query it but only attach a custom select over it.